pkg/kubectl/cmd/taint.go

```diff
@@ -80,7 +80,13 @@ var (
 		kubectl taint nodes foo dedicated:NoSchedule-
 
 		# Remove from node 'foo' all the taints with key 'dedicated'
-		kubectl taint nodes foo dedicated-`))
+		kubectl taint nodes foo dedicated-
+
+		# Add from node 'foo' all the taints with key 'dedicated'
+		kubectl taint nodes foo dedicated-
```

Comment:
It seems incorrect, the format with hyphen in the end ("dedicated-") is only for deletion. It does not make sense with addition.